src/index.html

```diff
@@ -1,12 +1,377 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="moyo-header">
+
+      <a href="#home" class="moyo-header__logo">
+        <img src="./images/logo.png" alt="Moyo logo">
+      </a>
+
+      <nav class="moyo-header__nav">
+        <a class="moyo-header__nav-link moyo-header__nav-link--is-active is-active" href="http://">Apple</a>
+        <a class="moyo-header__nav-link" href="http://">Samsung</a>
```

Comment:
```suggestion
        <a class="moyo-header__nav-link" href="#">Samsung</a>
```

Fix this everywhere to not open the error page on the link click